macos/Podfile.lock

```diff
@@ -1,15 +1,21 @@
 PODS:
   - FlutterMacOS (1.0.0)
+  - path_provider_macos (0.0.1):
```

Comment:
@klochowicz would you have any idea why this file and `macos/Runner.xcodeproj/project.pbxproj` was updated? It was not my intention to do so. Should these files actually be checked in?

Reply:
I think this is just the new flutter & xcode. happened to me as well. I believe that  we should have these files checked in so that it's reproducible.

Reply:
if possible, let's just extract them in a separate commit in this PR.